contracts/tasks/helpers/constants.ts

```diff
@@ -60,12 +45,7 @@ export const getNetworkRpcUrls = (): Record<ESupportedChains, string> => {
 
   return {
     [ESupportedChains.Sepolia]: `https://sepolia.infura.io/v3/${INFURA_KEY}`,
```

Comment:
So users won't use it. It's only for our internal usage. In case if we decide to get rid of cli deploy function we can add tweaks for RPC urls. But for now it seems fine just to have one provider key. Let me know if you have concerns about it.

Reply:
Sure let's keep it this way